eastdetector/__init__.py

```diff
@@ -228,3 +204,9 @@ def detect(self, image: np.ndarray, params=TextDetectorParams()) -> Union[None,
         boxes[:, :, 1] /= ratio_h
 
         return boxes
+
+
+    def close(self):
```

Comment:
I think there is no need to have this function, as this class does not inherit from our Detector interface